recipe/meta.yaml

```diff
@@ -57,9 +57,11 @@ outputs:
         - numpy
         - pillow >=4.1.1
         - pytorch =*={{ torch_proc_type }}*
+        - pytorch >=1.8.0
```

Comment:
Is it not possible to combine this into `pytorch >=1.8.0*={{ torch_proc_type }}*`?